test/integration/util/util.go

```diff
@@ -34,9 +36,9 @@ type MinishiftRunner struct {
 	CommandArgs string
 }
 
-func (m *MinishiftRunner) RunCommand(command string) (stdOut string, stdErr string, exitCode int) {
+func runCommand(command string, commandPath string) (stdOut string, stdErr string, exitCode int) {
```

Comment:
hmm, I'd probably wrap this into a struct similar to _MinishiftRunner_, just _OcRunner_ and then I let me give _MinishiftRunner_ an instance of _OcRunner_.